### ISSUE TITLE
Items are not clearly visible during drag

### ISSUE THREAD

**Issue by rootpox** [fatlinesofcode/ngDraggable#177]
Dragging items are not clearly visible as shown in attached screenshot. I have tried making  `[ng-drag].dragging {
  position: relative;
}` and `[ng-drag].dragging {
  position: absolute;
}` but it didn't work. I also noticed an issue ([#103](https://github.com/fatlinesofcode/ngDraggable/issues/103)) been reported and marked as closed. Could you please advice how to fix this?

![187240446cab335a683112194704f751](https://cloud.githubusercontent.com/assets/6086286/9105128/4faf7d38-3c32-11e5-9c30-203e93eed0ef.gif)

**Comment by dbounds** [comment]
Yes, I just upgraded from an earlier version and it appears the lib is now using transform3D when dragging. This appears to override the z-index.

I've been researching but have been unable to identify a fix.

**Comment by dbounds** [comment]
Additionally the solution referenced in #103 doesn't solve this for me.

**Comment by fatlinesofcode** [comment]
set allow-transform='false'

**Comment by Janjuks** [comment]
@fatlinesofcode still doesn't fix it

**Comment by Janjuks** [comment]
To be more precise:

If `data-allow-transform` is not present, then `z-index` is set to `9999` .. and element transofrmation is enabled, but `z-index` doesn't seem to work. 
When the `data-allow-transform` is set to `false`, there are no transformations, but also `z-index` isn't set. It works if you set it manually. But there is another problem - when setting it to false, element doesn't occupy original space anymore.

**Comment by hardiksondagar** [comment]
Setting position:absolute did the trick
`[ng-drag].dragging` {
position: absolute;
}`
